tests/virt_autotest/reboot_and_wait_up.pm

```diff
@@ -42,7 +42,7 @@ sub reboot_and_wait_up {
         #login
         #The timeout can't be too small since autoyast installation
         #need to wait 2nd phase install to finish
-        assert_screen("tty1-selected", 600);
+        assert_screen [qw(ttys1-selected tty1-selected)], 600;
```

Comment:
This looks like the wrong approach to me. Test modules should not care about the backend the tests are executed on. I understand there are backends where it is not tty1 but ttyS1. We should look for a more abstract tag describing what we want to achieve, not what a backend needs. How about `first-tty-selected`?

Reply:
okay, no problem, I can remove needle with ttys1-selected and create needle with tag first-tty-selected.

Reply:
So, that would be:

## Needles

Filename: tty1-selected-$date.*
Tag: first-tty-selected

Filename: ttyS1-selected-$date.*
Tag: first-tty-selected

## Code

```perl
assert_screen("first-tty-selected", 600);
```

Reply:
done:

https://openqa.suse.de/tests/3863221#step/reboot_and_wait_up_normal/4

Reply:
This is only one needle for all cases, which is the same that happened before with text-login, but with just another tag.
The problem that the previous PR was trying to solve is not solve with this PR, but at least with this PR other people will not be blocked. And the proper needles can be created later.